iFixit/react-commerce
# Product list page seo improvements

This PR closes #60 

## QA

1. Visit the Vercel PR preview
2. Visit the parts product list page
3. Verify that todos in #60 are done
